### PR TITLE
Bugfix in DCMTrajectoryGeneratorHelper::computeFeetWeight() function

### PR DESCRIPTION
**Pull request by GiulioRomualdi**
This PR fixes a bug in the weight evaluation in the `DCMTrajectoryGeneratorHelper` class
It also fixes the indentation in the aforementioned function
cc @bemilio

### CODE REVIEW

**Comment by bemilio** [comment]
For reference, here's a sample output after the fix. It makes sense to me.

weights_feet

File: src/DCMTrajectoryGeneratorHelper.cpp [hunk 1]
```diff
@@ -435,82 +435,81 @@ bool DCMTrajectoryGeneratorHelper::computeFeetWeight(const std::vector<StepPhase
...
+                feetDistance = iDynTree::toEigen(rightFootPosition) - iDynTree::toEigen(leftFootPosition);
 
-                    m_weightInLeft[instant] = ZMPDistanceFromLeftFoot.norm() / feetDistance.norm();
-                    m_weightInRight[instant] = 1 - ZMPDistanceFromLeftFoot.norm() / feetDistance.norm();
-                    ++instant;
-                }
+                ZMPDistanceFromLeftFoot = iDynTree::toEigen(zmpPosition[instant]) - iDynTree::toEigen(leftFootPosition);
 
+                m_weightInLeft[instant] = std::abs(1 - ZMPDistanceFromLeftFoot.norm() / feetDistance.norm());
+                m_weightInRight[instant] = std::abs(ZMPDistanceFromLeftFoot.norm() / feetDistance.norm());
```
Comment (S-Dafarra) [inline src/DCMTrajectoryGeneratorHelper.cpp]:
```suggestion
                m_weightInLeft[instant] = std::min(1, std::max(0.0, 1 - ZMPDistanceFromLeftFoot.norm() / feetDistance.norm())));
                m_weightInRight[instant] = 1.0 - m_weightInLeft[instant];
```

To make sure that all the values are between 0 and 1 and they sum up to 1.

Comment (GiulioRomualdi) [reply src/DCMTrajectoryGeneratorHelper.cpp]: Applied in d06e9843e77f097e6cda147e7ee15f1b727b620b

**Comment by GiulioRomualdi** [comment]
@S-Dafarra let me know if we can merge the PR

**Pull request merged**